CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 #### Minor
 
+- Added wrapper API endpoint to query a single contract agreement
```

Comment:
nit: depending on how much we add here, it may help to group changelog entries per component, for example

```suggestion
- API Wrapper:
    - Added wrapper API endpoint to query a single contract agreement
```

Reply:
And actually we could just always do it for easier changelog searching.